_docs/_latest/develop/test/integration-testing/create-test.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Create Tests
+order: 044
+---
+
+# {{ page.title }}
+
+This page describes how to add integration tests for your Forseti contributions.
+
+---
+## **Overview**
+
+The tech stack used to run Forseti integration tests consists of Kitchen, Kitchen 
+Terraform, InSpec and Google Cloud Build.
+
+[Kitchen](https://kitchen.ci/) provides a test harness to execute infrastructure 
+code on one or more platforms in isolation. It supports many testing frameworks 
+out of the box including InSpec.
+
+[Kitchen-Terraform](https://github.com/newcontext-oss/kitchen-terraform) is an 
+open source set of Test-Kitchen plugins for testing Terraform configuration. It 
+is a tool that lets us test Terraform modules using InSpec. 
+
+[InSpec](https://www.inspec.io/) is a free and open-source framework for testing 
```

Comment:
While the majority of the tests are currently in InSpec, I am working on moving them all over to pytest.